### PR TITLE
[KDESKTOP-801]  Shortens translations of 'Remove all synchronizations' 

### PR DESCRIPTION
**Pull request by luc-guyot-infomaniak**
The text for synchronizations removal is oversized and doesn't fit with the containing button size:

supprimer_toutes_les_syncs

This pull-request suggests shorter texts for the three supported languages distinct from english.

### CODE REVIEW

File: translations/client_de.ts [hunk 1]
```diff
@@ -624,7 +624,7 @@ Wählen Sie diejenigen aus, die Sie synchronisieren möchten:</translation>
     <message>
         <location filename="../src/gui/clientgui.cpp" line="1196"/>
         <source>REMOVE ALL SYNCHRONIZATIONS</source>
-        <translation>ALLE SYNCHRONISATIONEN ENTFERNEN</translation>
+        <translation>ALLE SYNC. ENTFERNEN</translation>
```
Comment (luc-guyot-infomaniak) [inline translations/client_de.ts:627]: I am hesitant: "ALL SYNCS ENTFERNEN" seems to be another viable choice.

Comment (herve-er) [reply translations/client_de.ts:627]: I don't speak Deutsch well, but DeepL and ChatGPT both seem to suggest:
> ALL SYNCS ENTFERNEN

Comment (herve-er) [reply translations/client_de.ts:627]:
```suggestion
        <translation>ALLE SYNCS ENTFERNEN</translation>
```

**Review by herve-er — COMMENTED** [review]
Could be interesting to also edit es version : 
image

File: translations/client_de.ts (hunk 1 above)
Comment (ClementKunz) [reply translations/client_de.ts:627]: Sounds like a solid argument in favor of "Sync." 👍

**Comment by luc-guyot-infomaniak** [comment]
> Could be interesting to also edit .es version : image

Thanks! It's of course required. Doing it in the next commit.

**Review by ClementKunz — APPROVED** [review]

**Pull request merged**